vllm/model_executor/layers/attention/paged_attn.py

```diff
@@ -0,0 +1,138 @@
+from typing import List, Optional
+
+import torch
+
+from vllm._C import cache_ops
+from vllm._C import ops
+from vllm.model_executor.input_metadata import InputMetadata
+from vllm.model_executor.layers.triton_kernel.prefix_prefill import (
+    context_attention_fwd)
+
+# Should be the same as PARTITION_SIZE in `paged_attention_v2_launcher`.
+_PARTITION_SIZE = 512
+
+
+class PagedAttention:
```

Comment:
Call this class `PagedAttentionImpl` or something like `PagedAttentionWrapper`? I think it's better to make it more clear that this is not a PyTorch module class.